debezium-server/debezium-server-pulsar/pom.xml

```diff
@@ -66,16 +66,15 @@
             <artifactId>kafka-connect-avro-converter</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.testcontainers</groupId>
-            <artifactId>testcontainers</artifactId>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>io.debezium</groupId>
             <artifactId>debezium-connector-postgres</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
```

Comment:
Should remain in "test" scope?

Reply:
As `PulsarContainer` is under the main dir (`debezium/debezium-server/debezium-server-pulsar/src/main/java/io/debezium/server/pulsar/PulsarContainer.java`), hence I removed the `scope`.

Reply:
Oh, that's interesting. `PulsarContainer` should be a test source.